projects/npm-tools/packages/changelog-generator/src/index.js

```diff
@@ -749,6 +937,41 @@ async function main(_node, _script, ...args) {
 		}
 	}
 
+	try {
+		await go(options);
```

Comment:
This is an example of a thing to refactor. The name `go` sucks; I just need something so that I could have `--interactive` mode run once as a preview, and then recurse to run again "for realz". I'll see if I can find something cleaner.